src/contribute/contribution-guide.md

```diff
@@ -218,6 +218,26 @@ push. You'll run:
 
 	$ git push <GitHub_user> <my-branch> --force
 
+### Building
+
+#### Python SDK
+
+Before testing SDK code changes remotely, the Beam tarball has to be built. From
```

Comment:
"the Beam tarball has to be built." -> "you must build the Beam tarball."

Reply:
done